package.json

```diff
@@ -30,9 +30,10 @@
     "prepublishOnly": "echo '***\nUse `npm run clean-publish` to publish govuk-prototype-kit\n***' && exit 1",
     "tmp-kit": "mkdir -p $TMPDIR/govuk-prototype-kit-playground && cd $TMPDIR/govuk-prototype-kit-playground && rm -Rf ./* && govuk-prototype-kit create --version local . && npm run dev",
     "start": "echo 'This project cannot be started, in order to test this project please create a prototype kit using the cli.'",
-    "start:package": "node cypress/scripts/run-starter-prototype",
-    "start:package:prodtest": "node cypress/scripts/run-starter-prototype --prodtest",
-    "start:package:heroku": "cross-env USE_AUTH=false USE_HTTPS=false node scripts/create-prototype-and-start 'npx --yes heroku local --port 3000'",
+    "start:dev": "node scripts/create-prototype-and-run",
```

Comment:
what about just `npm run dev` like prototypes?

Reply:
Personally I feel it's helpful to have all the different start scripts grouped together like this, but that's just my opinion.

Reply:
I agree with Laurence @lfdebrux, It's better that the automated tests are grouped as it's easy to see that they are specific for running the prototype in acceptance tests